pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -203,6 +207,27 @@ public void setNumSegmentsMatched(long numSegmentsMatched) {
     _numSegmentsMatched = numSegmentsMatched;
   }
 
+  @JsonProperty("numConsumingSegmentsQueried")
+  public long getNumConsumingSegmentsQueried() {
+    return _numConsumingSegmentsQueried;
+  }
+
+  @JsonProperty("numConsumingSegmentsQueried")
+  public void setNumConsumingSegmentsQueried(long queried) {
```

Comment:
(Optional) change the parameter to be **numConsumingSegmentsQueried** (follow the convention of other methods)